spec/feature/user_login_spec.rb

```diff
@@ -19,9 +19,6 @@
 
   describe 'SHOW' do
     it 'shows user' do
-      admin = User.where(first_name: 'Admin').first
-      visit("users/#{admin.id}")
-      expect(page).to have_content('Admin User')
```

Comment:
Why did you remove this? Doesn't this just remove the test entirely?

Reply:
Unintended commit, meant to revert the file after I saw my tests passing.

Reply:
Does this mean I'm good to just put the lines back in?

Reply:
Yeah, this change can be reverted